homework/index.js

```diff
@@ -29,25 +31,113 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function changeDateFormat(date) {
+    const timeFormat = new Date(date);
+    return timeFormat.toLocaleString();
+  }
+
+  function createTableRow(table, header, content) {
+    const row = createAndAppend('tr', table, { class: 'row' });
+    createAndAppend('th', row, { text: header, class: 'table-header' });
+    createAndAppend('td', row, { text: content, class: 'content' });
+    return row;
+  }
+
+  function renderRepoDetails(repo, repoName) {
```

Comment:
This second parameter does not expect the name of the repo. It expect the container (parent) element in which to render the repo details. Please make an effort to be precise in your naming. Otherwise you are confusing the people trying to understand your code.